src/Azure.Functions.Cli/ExtensionBundle/ExtensionBundleHelper.cs

```diff
@@ -32,7 +33,7 @@ public static ExtensionBundleManager GetExtensionBundleManager()
                 extensionBundleOption.DownloadPath = GetBundleDownloadPath(extensionBundleOption.Id);
                 extensionBundleOption.EnsureLatest = true;
             }
-            return new ExtensionBundleManager(extensionBundleOption, SystemEnvironment.Instance, NullLoggerFactory.Instance);
+            return new ExtensionBundleManager(extensionBundleOption, SystemEnvironment.Instance, NullLoggerFactory.Instance, new FunctionsHostingConfigOptions());
```

Comment:
The build was failing, upon investigation we found that `ExtensionBundleManager` constructor now requires an additional non-optional parameter of `FunctionsHostingConfigOptions` object.

Reply:
We probably need approval from someone from host team before we merge this PR.